src/AddProjectsPage/AddProjectsPage.tsx

```diff
@@ -71,9 +74,13 @@ class AddProjectsPage extends React.Component<PassedProps, State> {
   }
 
   render() {
+    let isUserLoggedIn = false;
+    if (this.props.user.email) {
+      isUserLoggedIn = true;
+    }
     return (
       <div>
-        <Header />
+        {isUserLoggedIn === true ? <LoggedInHeader /> : <Header />}
```

Comment:
Line 83, this code is repeated in all the files that require a header. Is there a way to make it so the conditional is used once in the project? And the correct header is passed to all the pages requiring a header?